src/main/xsd/state/1.0.0/state.xsd

```diff
@@ -1681,6 +1681,15 @@
         </xs:annotation>
       </xs:element>
 
+      <xs:element name="slices" type="fade:SlicesType"
+        minOccurs="0" maxOccurs="1">
+        <xs:annotation>
+          <xs:documentation xml:lang="en" xml:space="preserve">
+            One or more time slice intervals.
```

Comment:
The documentation should explain the behavior of slice intervals. Maybe something along the lines of:
```
Defines an interval less than 24 hours in which data is loaded for each day in the load interval.
```
Feel free to spruce that up if it seems unclear.

Reply:
Added revised docs here, and added missing xml annotations for new types.